v2.1/cancel-query.md

```diff
@@ -26,6 +26,7 @@ The `root` user can cancel any currently active queries, whereas non-`root` user
 Parameter | Description
 ----------|------------
 `query_id` | A [scalar expression](scalar-expressions.html) that produces the ID of the query to cancel.<br><br>`CANCEL QUERY` accepts a single query ID. If a subquery is used and returns multiple IDs, the `CANCEL QUERY` statement will therefore fail.
```

Comment:
I would go with "the ... statement will fail", the "therefore" is unnecessary IMO.

Reply:
It seems like this should be updated with the same text as the note below, re: using `CANCEL QUERIES` in the case where there are multiple query IDs returned.  Or am I misunderstanding what I'm seeing?